recipes/genie/meta.yaml

```diff
@@ -0,0 +1,30 @@
+{% set version = "0.4" %}
+
+package:
+  name: genie
+  version: {{ version }}
+
+source:
+  url: https://github.com/sakkayaphab/genie/releases/download/v{{ version }}/genie_darwin_amd64.tar.gz # [osx]
```

Comment:
Can you please try to compile this tool?

Reply:
I've recompiled

Reply:
Björn meant compile it in the recipe, rather than packaging a binary.

Reply:
Sorry i misunderstood, but now I have fixed